front-end/src/Main/Services/Reducers/reducers.js

```diff
@@ -0,0 +1,24 @@
+import * as types from '../Actions/types';
+
+const initialState = {
+  userType: 'guest',
+  isAuth: false,
+  isReady: true,
+};
+
+const app = (state = initialState, action) => {
+  switch (action.type) {
+    case types.IS_LOGGED_IN: {
```

Comment:
Confilicts with Julia code can appears...

Reply:
This part of code is not finished yet. I was waiting for Julia's pull request to be approved, so i could finish what I wanted to do.